infra/gcp/README.md

```diff
@@ -40,12 +40,12 @@ infrastructure.
 - Go to the Google Cloud console, API & Services.
   - Configure the consent screen.
     - You can probably leave most fields on the first page empty. Give it a sensible name and management email.
-    - Add the scope: `../auth/userinfo.email`.  
+    - Add the scope: `../auth/userinfo.email`.
```

Comment:
Seems like a lot of noisy changes in this file. Which is fine, I guess... but you will be on the git blame forever now

Reply:
My editor settings strip trailing whitespace. I usually try to be better about not having these superfluous changes, but like you said, it's fine.